yammbs/torsion/_store.py

```diff
@@ -587,3 +590,253 @@ def get_metrics(
             }
 
         return metrics
+
+    def get_proper_torsion_by_torsion_id(
```

Comment:
I thought some about making this private since it's only used as a helper function here, but I suspect it could be useful in other cases so I lean towards keeping it public

Reply:
OK, great. I've given this a more informative name: `get_proper_torsion_parameters_by_torsion_id`.